l10n_nl_tax_statement_icp/models/l10n_nl_vat_statement_icp_line.py

```diff
@@ -51,11 +51,19 @@ def _compute_icp_amount_format(self):
 
     @api.constrains('country_code')
     def _check_country_code(self):
-        country_codes = self.mapped('country_code')
+        europe_codes = self.env.ref('base.europe').country_ids.mapped('code')
+        if "GB" not in europe_codes:
```

Comment:
I think [this part](https://github.com/OCA/l10n-netherlands/blob/b7babb2219f8a765d06d9a37e367141cffab9245/l10n_nl_tax_statement_icp/models/l10n_nl_vat_statement_icp_line.py#L68-L71) will raise an error in case GB is not in europe_codes.

Reply:
That part raises an error if 'GB' ends up in country_codes, at least if GB is not a member of the European countries group (which it should not be, by now). So what I'm doing is I'm filtering out lines with code 'GB' from before the brexit date. If there are no GB lines from after brexit, 'GB' will not be in country codes and no error is raised.

Reply:
But when you do `or line.statement_id.from_date >= brexit_date` you expect lines with "GB", right?

Reply:
Yes, GB lines from after brexit_date will be included in `lines` so that the validation will be applied to them.

Reply:
Shouldn't the raise be skipped in that case?

Reply:
I was aiming to skip the raise for GB lines from *before* the brexit date, not after. I should add tests to demonstrate this, shoudn't I?

Reply:
Sorry, my mistake. You check that `line.statement_id.from_date >= brexit_date` so it is ok as it is.